google-cloud-core/src/com/google/cloud/tools/intellij/analytics/GctTracking.java

```diff
@@ -105,4 +105,5 @@
   public static final String MANAGED_SDK_SUCCESSFUL_UPDATE = "managed.sdk.successful.update";
   public static final String MANAGED_SDK_FAILED_UPDATE = "managed.sdk.failed.update";
   public static final String MANAGED_SDK_CANCELLED_UPDATE = "managed.sdk.cancelled.update";
+  public static final String MANAGED_SDK_VERSION_PARSE_ERROR = "managed.sdk.version.parse.error";
```

Comment:
Will that apply to custom SDK as well?

Reply:
oh yeah. It will. I'll just rename the variable. that should cover it I think

Reply:
done

Reply:
great thanks!